src/test/kotlin/de/klg71/keycloakmigration/changeControl/actions/identityprovider/AddSamlEmailAddressAttributeMapperIntegTest.kt

```diff
@@ -0,0 +1,55 @@
+package de.klg71.keycloakmigration.changeControl.actions.identityprovider
+
+import de.klg71.keycloakmigration.AbstractIntegrationTest
+import de.klg71.keycloakmigration.keycloakapi.KeycloakClient
+import de.klg71.keycloakmigration.keycloakapi.identityProviderByAlias
+import de.klg71.keycloakmigration.keycloakapi.identityProviderMapperByName
+import de.klg71.keycloakmigration.keycloakapi.model.SAML_ATTRIBUTE_EMAILADDRESS
+import de.klg71.keycloakmigration.keycloakapi.model.SAML_USER_ATTRIBUTE_IDP_MAPPER
+import org.assertj.core.api.Assertions
+import org.junit.Test
+import org.koin.core.component.inject
+
+class AddSamlEmailAddressAttributeMapperIntegTest : AbstractIntegrationTest() {
+
+    val client by inject<KeycloakClient>()
+
+    @Test
+    fun testAddSamlEmailAddressAttributeMapper() {
```

Comment:
I know it is boring and lame but I would really appreciate a test for the rollback method too

Reply:
Good point. I've added the respective tests.